ducktape/services/service.py

```diff
@@ -150,6 +150,12 @@ def wait(self):
         """
         pass
 
+    def wait_node(self, node, timeout_sec=None):
```

Comment:
Do we really want this in the base service class? I was imagining it would belong in `BackgroundThreadService`